src/dual.jl

```diff
@@ -410,6 +410,38 @@ end
 @ambiguous @inline Base.atan2(y::Real, x::Dual) = calc_atan2(y, x)
 @ambiguous @inline Base.atan2(y::Dual, x::Real) = calc_atan2(y, x)
 
+@inline function Base.fma(x::Dual, y::Dual, z::Dual)
+    vx, vy = value(x), value(y)
+    result = fma(vx, vy, value(z))
+    return Dual(result,
```

Comment:
Wouldn't it be better to define a `fma` function for `Partials` as well, so that `fma` instructions will be used?